### ISSUE TITLE
proto 3 syntax support

### ISSUE THREAD

**Issue by develar** [dcodeIO/ProtoBuf.js#252]
Valid proto file 

```
syntax = "proto3";
message Point {
  int32 latitude = 1;
  int32 longitude = 2;
}
```

is not parsed by pbjs: Illegal token in message Point at line 3: int32

I use ProtoBuf.js-4.0.0-b1.

**Comment by cfallin** [comment]
Just to avoid duplication of work: I have a patch for this and I'm working on getting it approved so I can submit a PR. Hopefully soon!

**Comment by cfallin** [comment]
Included in PR #254.
